API/models/model.go

```diff
@@ -786,15 +809,21 @@ func UpdateObject(entityStr string, id string, updateData map[string]interface{}
 		}
 
 		if oldObj["id"] != updateData["id"] {
-			// Changes to id should be propagated to its children
-			err := PropagateParentIdChange(
+			// Changes to id should be propagated
+			if err := PropagateParentIdChange(
 				ctx,
 				oldObj["id"].(string),
 				updateData["id"].(string),
 				entity,
-			)
-			if err != nil {
+			); err != nil {
 				return nil, err
+			} else if entity == u.DOMAIN {
```

Comment:
The else is not necessary since there is a return inside the if

Reply:
I just prefer it for the readability